src/proto/NEO.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package TW.NEO.Proto;
+option java_package = "wallet.core.jni.proto";
+
+message TransactionInput {
+    bytes prev_hash = 1;
+    fixed32 prev_index = 2;
+}
+
+message TransactionOutput {
+    string asset_id = 1;
+    sint64 value = 2;
+    string address = 3;
+}
+
+// Input data necessary to create a signed transaction.
+message SigningInput {
+    repeated TransactionInput inputs = 1;
+    repeated TransactionOutput outputs = 2;
+    bytes private_key = 3;
```

Comment:
Sorry for overlooked about fee and changes: 
can you take a look at Bitcoin.proto or Cardano.proto? how do we calculate fee? for other UTXO blockchains, we will do two phases sign:
1. calculate the fee and return a TransactionPlan
2. the signer accept the plan and generate output automatically

Reply:
I added the TransactionPlan implementation as in Cardano, but with some corrections for NEO. The fact is, you can transfer NEO or several different tokens at once, but the fee must be paid in GAS.

> Transactions larger than 1024 bytes require 0.001 GAS as a base fee, in addition to 0.00001 GAS per byte of the transaction. Most basic NEP-5 transfers remain well under the 1024 byte threshold, however transactions with more complex logic or large quantities of inputs and outputs may exceed the threshold and require a network fee.